### PR TITLE
Reuse branch name

### PR DESCRIPTION
**Pull request by gchomatas**

### CODE REVIEW

**Comment by jhaber** [comment]
When a branch is deleted, GitHub sends a deletion event but they also send a push event. In the past, depending on the order the events are processed, this push event would accidentally reactivate the branch. Does this PR handle that case?

File: BlazarService/src/main/java/com/hubspot/blazar/util/GitHubWebhookHandler.java [hunk 1]
```diff
@@ -106,11 +108,7 @@ public void handlePushEvent(PushEvent pushEvent) throws IOException {
 
     if (!pushEvent.getRef().startsWith("refs/tags/") && !pushEvent.getDeleted()) {
```
Comment (Jw0x47) [inline BlazarService/src/main/java/com/hubspot/blazar/util/GitHubWebhookHandler.java:109]: @jhaber The check for `.getDeleted()` here prevents us from re-activating branches that were deleted.

**Comment by jonathanwgoodwin** [comment]
:ship

**Comment by Jw0x47** [comment]
🚢

File: BlazarService/src/main/java/com/hubspot/blazar/util/GitHubWebhookHandler.java (hunk 1 above)
Comment (jhaber) [reply BlazarService/src/main/java/com/hubspot/blazar/util/GitHubWebhookHandler.java:109]: Ah interesting

**Pull request merged**